fl4health/clients/fenda_client.py

```diff
@@ -46,13 +49,16 @@ def __init__(
             Each value associate with one of two contrastive losses in perfcl loss.
             cos_sim_loss_weight: Weight to be used for cosine similarity loss.
             contrastive_loss_weight: Weight to be used for contrastive loss.
+            mkmmd_loss_weight: Weight to be used for mkmmd loss.
         """
         self.perfcl_loss_weights = perfcl_loss_weights
         self.cos_sim_loss_weight = cos_sim_loss_weight
         self.contrastive_loss_weight = contrastive_loss_weight
-        self.cos_sim = torch.nn.CosineSimilarity(dim=-1).to(self.device)
-        self.ce_criterion = torch.nn.CrossEntropyLoss().to(self.device)
+        self.mkmmd_loss_weight = mkmmd_loss_weight
         self.temperature = temperature
```

Comment:
Since temperature is just being used to initialize the ContrastiveLoss class, I don't think we need to keep it around as a class variable?

Reply:
same here